digital-twins/jsonld-contexts/digital_twins_compound.jsonld

```diff
@@ -1,6 +1,11 @@
 {
   "@context": [
     "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/usecase.jsonld",
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/airQualityMonitoring/jsonld-contexts/airQualityMonitoring.jsonld",
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/irrigation/jsonld-contexts/irrigation.jsonld",
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/AlertAnomalyRisk/jsonld-contexts/AlertAnomalyRisk-compound.jsonld",
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/alertAndAnomaly/jsonld-contexts/alertAndAnomaly.jsonld",
```

Comment:
seems weird

Reply:
yeah not sure which one I need more but I prefer have all of them so I don't need to update it later in case

Reply:
it can also cause unwanted side effects (terms conflict / overriding, ...). not always a good thing.

Reply:
Ok so I'll keep just the most recent